Java/Java.sublime-syntax

```diff
@@ -227,7 +338,15 @@ contexts:
                 - match: "}"
                   pop: true
                 - include: class-body
+
   keywords:
+    - match: '::'
```

Comment:
This should probably be either `punctuation.accesor` (that would be for more "color" consistency at the expense of semantic correctness), or `keyword.operator` for semantic correctness, correct?